docs/baking.md

```diff
@@ -0,0 +1,175 @@
+
+In order to import the snapshot run the following command:
+```
+sudo -u tezos tezos-node snapshot import --data-dir /var/lib/tezos/node-<network>/ <path to the snapshot file>
+```
+
+#### Troubleshooting node snapshot import
+
+Snapshot import requires that the `tezos-node` data directory doesn't contain the `context` and `store` folders.
+Remove them in case you're getting such error about their presence.
+
+In case you're getting an error similar to
+```
+tezos-node: Error:
+              Invalid block BLjutMj47caB
+                Failed to validate the economic-protocol content of the block: Error:
+                                                                                Invalid signature for block BLjutMj47caB. Expected: tz1Na5QB98cDA.
+```
+
+You should init/update config in the data directory to match the desired network
+before importing the snapshot. To do that run one of the following commands
+depending on whether the config in the given data directory was initialized previously:
+```
+sudo -u tezos tezos-node config init --data-dir /var/lib/tezos/node-<network> --network <network>
+#or
+sudo -u tezos tezos-node config update --data-dir /var/lib/tezos/node-<network> --network <network>
+```
+
+### Starting the node
+
+After the snapshot import, you can finally start the node by running
+```
+sudo systemctl start tezos-node-<network>.service
+```
+
+Note that even after the snapshot import the node can still be out of sync, it may require
+some additional time to completely bootstrap. In order to check whether the node is bootstrapped,
+you can use `tezos-client`:
+```
+tezos-client --endpoint <node-rpc-address> bootstrapped
+```
+
+To stop the node run:
+```
+sudo systemctl stop tezos-node-<network>.service
+```
+
+You can check the node logs via `journalctl`:
+```
+journalctl -u tezos-node-<network>.service
+```
+
+## Setting up baker and endorser daemons
+
+### Setting up daemons data directory
+
+Data directories for baker and endorser daemons are defined in the
+`/etc/default/tezos-baker-<proto>` and `/etc/default/tezos-endorser-<proto>`.
+By default, both this daemons have `/var/lib/tezos/client` set as a `DATA_DIR`.
+
+Additionally, you should specify `NODE_DATA_DIR` in the `/etc/default/tezos-baker-<proto>`
+to point at the desired node data directory, e.g. `/var/lib/tezos/node-<network>`.
+
+
+### Importing baker key
+
+Import your baker secret key to the data directory. There are two ways to import
+such key:
+1) You know either the unencrypted or password-encrypted secret key for your address.
+
+In order to import such key run:
+```
+sudo -u tezos tezos-client -d /var/lib/tezos/client import secret key <alias> <secret-key>
+```
+2) The secret key is stored on a ledger.
+
+Open the Tezoz Wallet app on your ledger and run the following
+to import the key:
+```
+sudo -u tezos tezos-client -d /var/lib/tezos/client import secret key <alias> <ledger-url>
```

Comment:
This and the one just above also could be used without the `-d` argument if we change the default config